src/atomate2/vasp/flows/defect.py

```diff
@@ -22,27 +33,160 @@
 
 logger = logging.getLogger(__name__)
 
+# Defaults
+DUMMY_STRUCT = Structure(Lattice.cubic(3.6), ["Si", "Si"], [[0.5, 0.5, 0.5], [0, 0, 0]])
+
+SPECIAL_KPOINT = Kpoints(
+    comment="special k-point",
+    num_kpts=1,
+    style=Kpoints_supported_modes.Reciprocal,
+    kpts=((0.25, 0.25, 0.25),),
+    kpts_shift=(0, 0, 0),
+    kpts_weights=[1],
+)
+
+SPECIAL_KPOINT_GAMMA = Kpoints(
+    comment="special k-point",
+    num_kpts=1,
+    style=Kpoints_supported_modes.Reciprocal,
+    kpts=((0.25, 0.25, 0.25), (0.25, 0.25, 0.25)),
+    kpts_shift=(0, 0, 0),
+    kpts_weights=[1, 0],
+)
 
-DEFAULT_DISTORTIONS = (-1, -0.15, -0.1, -0.05, 0, 0.05, 0.1, 0.15, 1)
-DEFECT_INCAR_SETTINGS = {
-    "ISMEAR": 0,
-    "LWAVE": True,
-    "SIGMA": 0.05,
-    "KSPACING": None,
-    "ENCUT": 500,
-}
-DEFECT_KPOINT_SETTINGS = {"reciprocal_density": 64}
+# Default relax should use: PBE -> HSE to save computation time
+PBE_GEN: AtomicRelaxSetGenerator = AtomicRelaxSetGenerator(
```

Comment:
In general, I'd like to avoid inheriting from subclasses of the input set generator. This is because it makes it very difficult to work out which settings have been changed as you have to go through all subclass layers.

I think we could add two new defect input sets, one for PBEsol and one for HSE. I suggest they are called `ChargeStateRelaxSetGenerator` and `HSEChargeStateRelaxSetGenerator` to be consistent with naming of the other input sets. They should live in `atomate2/vasp/sets/defect.py` file.